src/legacy/ui/public/inspector/build_tabular_inspector_data.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { set } from 'lodash';
+// @ts-ignore
+import { createFilter } from '../vis/vis_filters';
+import { FormattedData } from './adapters';
+
+interface Column {
+  id: string;
+  name: string;
+  aggConfig: any;
+}
+
+interface Row {
+  [key: string]: any;
+}
+
+interface Table {
+  columns: Column[];
+  rows: Row[];
+}
+
+/**
+ * @deprecated
+ *
+ * Do not use this function.
+ *
+ * @todo This function is used only by Courier. Courier will
+ *   soon be removed, and this function will be deleted, too. If Courier is no removed,
+ *   move this function inside Courier.
+ *
+ * ---
+ *
+ * This function builds tabular data from the response and attaches it to the
+ * inspector. It will only be called when the data view in the inspector is opened.
+ */
+export async function buildTabularInspectorData(table: Table, queryFilter: any) {
```

Comment:
can make it `unknown` instead of `any`?

```suggestion
export async function buildTabularInspectorData(table: Table, queryFilter: unknown) {
```

Reply:
Added more specific type.